sql/src/test/java/io/crate/executor/transport/TransportExecutorTest.java

```diff
@@ -610,7 +611,8 @@ public void testUpdateByIdTask() throws Exception {
                 new HashMap<String, Symbol>(){{
                     put(nameRef.info().ident().columnIdent().fqn(), Literal.newLiteral("Vogon lyric fan"));
                 }},
-                Optional.<Long>fromNullable(null)
+                Optional.<Long>fromNullable(null),
```

Comment:
not really your change but instead of `fromNullable(null)` it would be possible to use `Optional.absent()`